www/delivery_dev/async.js

```diff
@@ -39,19 +39,35 @@
                  * @param object data
                  */
                 ajax: function (url, data) {
-                    var xhr = new XMLHttpRequest();
+                    if ($.browser.msie && window.XDomainRequest) {
+                        var emptyFunction = function(){ return; },
+                            xdr = new XDomainRequest();
+
+                        xdr.onerror = emptyFunction;
+                        xdr.ontimeout = emptyFunction;
+                        xdr.onprogress = emptyFunction;
+                        xdr.onload = function () {
+                            rv.spc(JSON.parse(this.responseText));
```

Comment:
@mbeccati 

I have updated the variable name. You were right, it was coming from the minified version.

This one should be ok now.